scripts/upload-example-application-to-sauce-labs.sh

```diff
@@ -26,21 +26,23 @@ fi
 export SAUCE_USERNAME=$BUILDKITE_MAC_STADIUM_SAUCE_USERNAME
 export SAUCE_ACCESS_KEY=$BUILDKITE_MAC_STADIUM_SAUCE_ACCESS_KEY
 
-echo "▸ Uploading test application to Sauce Labs App Storage"
+# saucectl does these parts for us, we don't need to curl anything
```

Comment:
I think we still need this step because we do not declare our SL credentials in the config file.

Reply:
If we skip this step, we're still able to run the tests but we'll be using an example app as well as tests from a previous commit.